pyacq/core/stream.py

```diff
@@ -309,20 +338,24 @@ def __init__(self, socket, params):
         elif self.params['compression'] in ['blosc-blosclz', 'blosc-lz4']:
             self.func = self._uncompress_blosc
     
-    def recv(self):
+    def recv(self, autoswapaxes = True):
         m0,m1 = self.socket.recv_multipart()
         index = np.fromstring(m0, dtype='int64')[0]
```

Comment:
You can get a performance boost here by using `np.frombuffer` instead (it will avoid a copy).